src/cloud/live/stage-prod-test.md

```diff
@@ -40,53 +40,53 @@ Check the Magento configuration settings through the Admin panel including the B
 
--  Staging: `dig http[s]://staging.<your domain>.c.<instanceid>.ent.magento.cloud`
--  Production: `dig http[s]://<your domain>.{1|2|3}.<project ID>.ent.magento.cloud`
+1. Navigate to **Stores** > **Settings** > **Configuration** > **Advanced** > **System**. Scroll and click **Full Page Cache**.
 
-Next, use a `curl` command to verify X-Magento-Tags exist and additional header information. The format for the command is:
+1. Ensure that the **Caching application** value is set to _Fastly CDN_ .
 
-```bash
-curl http[s]://<full site URL> -H "host: <url>" -k -vo /dev/null -HFastly-Debug:1
-```
+1. Test the Fastly credentials.
 
-For Starter, enter the full site URL from your environment [Access info][] in the command to view the headers.
+   -  Click **Fastly Configuration**.
 
-For Pro Staging and Production, the command differs per server:
+   -  Verify that the values for the Fastly Service ID and Fastly API token credentials. See [Get Fastly credentials][].
 
--  Staging: `curl http[s]://staging.<your domain>.c.<instanceid>.ent.magento.cloud -H "host: <url>" -k -vo /dev/null -HFastly-Debug:1`
--  Production:
-   -  The load balancer: `curl http[s]://<your domain>.c.<project ID>.ent.magento.cloud -H "host: <url>" -k -vo /dev/null -HFastly-Debug:1`
-   -  A direct Origin node: `curl http[s]://<your domain>.{1|2|3}.<project ID>.ent.magento.cloud -H "host: <url>" -k -vo /dev/null -HFastly-Debug:1`
+   -  Click **Test credentials**.
 
-After you are live, you can also check your live site: `curl https://<your domain> -k -vo /dev/null -HFastly-Debug:1`. You can also add `--resolve` if your live URL is not set up with DNS.
+   {:.bs-callout-warning}
+   Make sure that you entered the correct Fastly Service ID and API token in your Staging and Production environments. Fastly credentials are created and mapped per service environment. If you enter Staging credentials in your Production environment, you cannot upload your VCL snippets, caching does not work correctly, and your caching configuration points to the wrong server and stores.
 
-Check the returned response headers and values:
+{:.procedure}
+To check Fastly caching behavior:
 
--  `Fastly-Magento-VCL-Uploaded` should be present
--  `X-Magento-Tags` should be returned
--  `Fastly-Module-Enabled` should be either `Yes` or the Fastly extension version number
--  `X-Cache` should be either `HIT` or `HIT, HIT`
--  `x-cache-hits` should be 1,1
--  [`Cache-Control: max-age`][] should be greater than 0
--  [`Pragma`][] should be `cache`
+1. Check for headers using the `dig` command-line utility to get information about the site configuration.
 
-To verify Fastly is enabled in Staging and Production, check the configuration in the Magento Admin for each environment:
+   The following examples use Pro URLs. You can use any URL with the `dig` command.
```

Comment:
Does it seem smoother if you state a fact and then draw their attention to the examples?

```suggestion
   You can use any URL with the `dig` command. The following examples use Pro URLs:
```